README.md

```diff
@@ -11,16 +11,39 @@ Image Template to build custom notebook images for JupyterHub.
 ## How to build the image
 
 ### Locally
+
+#### Prerequisites
+
+* **s2i**: We assume majority of users will be using linux and recommend using the the instructions
+[here](https://github.com/openshift/source-to-image#for-linux) to install s2i if needed.
+
+* **Podman**: Please follow the instructions [here](https://github.com/openshift/source-to-image#for-linux) for your
+linux distribution.
+
+* **Docker**: If you are using Fedora please use these [instructions](https://docs.docker.com/engine/install/fedora/)
+to install docker. If you are on RHEL8 you can run into issues with docker, please follow the additional documentation
+that can be found [here](docs/RHEL8_docker_install.md).
+
+#### Build
+
   You can build the image locally using s2i, by executing the following command
   in the root of your repository. <br>
-    `s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest`
+````
+$ s2i build . quay.io/thoth-station/s2i-custom-notebook:latest MY-CUSTOM-IMAGE:latest
+````
+Be sure to replace `MY-CUSTOM-IMAGE` above with the name you intend to use for your image.
 
 You might need to update your deployment of JupyterHub for the newly built/imported image to
 be available for use.
 
 # Running the image Locally
+
 After building the image, you can run it locally using the following command: <br>
-`podman run -p 8080:8080 MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080`
+```
+$ podman run -p 8080:8080 docker-daemon:MY-CUSTOM-IMAGE:latest start-singleuser.sh --ip="0.0.0.0" --port=8080
+```
+Since the s2i notebook builder relies on docker, and we prefer to use podman, make sure to prepend your image name with
+`docker-daemon:` so that podman checks the correct local registry.
```

Comment:
Can you add a line that says something like, "you can use `docker` instead of `podman` in the same command and it should work"?

Reply:
To address the docker podman confusion, I think here will give the docker run command first, then say something like, "if you prefer to use podman you can also do that by running this command..."  does that sound good?

Reply:
yeah sounds good :+1: